operations/app/terraform/modules/storage/main.tf

```diff
@@ -179,7 +179,7 @@ resource "azurerm_storage_account" "storage_public" {
   }
 
   network_rules {
-    default_action = "Allow"
+    default_action = "Deny"
```

Comment:
This is a defense-in-depth setting.  Since the web service does not appear to be affected, the only access path that could still be an issue is updating the web content.  Is there a way to test updating web content in test?